explainers/navigation-to-bundles.md

```diff
@@ -0,0 +1,83 @@
+
+* be shared and loaded in a self-contained way, similar to what packaged apps and other portable data representation formats provide,
+* be composed as a distributable, installable application, or
+* load reliably and quickly even when a site consists of a large number of subresources, and even when the internet access is limited.
+
+Instead, imagine if we could bundle up a “full” website in a single resource file, so that the website could be shared via a SD card or over some p2p protocol, or could be retrieved from a fast cache or a nearby proxy. It would open up various interesting use cases.
+
+**Web Bundle** is a proposal that tries to achieve it. It’s a format that can represent a collection of HTTP resources, and therefore can represent one or multiple web pages (which typically consist of multiple resources, e.g. HTML, javascripts, images and styles) in a single file.  It’s a part of [Web Packaging](https://github.com/WICG/webpackage) proposal and is also known as "**Bundled HTTP Exchanges** ([spec proposal](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html))."
+
+This document explains how a Web Bundle can be a navigation target, so that a user can open a given Web Bundle in browsers (UAs) and navigate into the website represented by the Bundle.
+
+## Example Scenarios: Save and browse an unsigned content
+UA can provide **‘Save as bundle’** feature that will dynamically generate an unsigned bundle of the page that a user is currently browsing.  Then anyone can use the feature to get a bundled representation of the current page, which can be browsed later by the same user, or can be shared with a nearby friend via some person-to-person sharing mechanism.
+
+When someone else browses the unsigned bundle, they see some notation in their URL bar that isn’t the URL of the original URL, but they can still browse around and see the JS executing.  Note that in this case the page coming from the unsigned bundle is not given any access to the cookie/storage of the original site.
+
+
+## Example Scenarios: Publishing a signed content
+The author of a site creates a bundle for one or a few pages of the site, signs the bundle with the site’s certificate, and then publish the bundle in the way that interested users can find it.
+
+Later, when a user finds the bundle, they can navigate their UA client to the bundle, maybe by opening it as a local file copied over SD card or by navigating to the distribution URL where the bundle is published.  The UA should be able to parse and verify the bundle’s signature, and then to navigate to the website represented by the bundle, without actually connecting to the site as all the necessary subresources could be served by the bundle.  If the bundle represents multiple pages for the site, the user should also be able to browser those pages without worrying about the connectivity.
+
+# Navigation
```

Comment:
```suggestion
# Details of how the browser handles navigation
```

I'm not doing a detailed review of the text below, because it's intrinsically more technical than we want to make an explainer's audience read. It'll eventually move into the specification.